Controls/Tests/Test_Cafe.c

```diff
@@ -21,14 +21,57 @@ int main(void){
 	OS_CPU_SysTickInit();
 
 	// Create any semaphores/mutexes and initialize any global variables here
+	//OS_MUTEX RegisterOccupied_Mutex;
 	
+	OSMutexCreate(
+		(OS_MUTEX*) &RegisterOccupied_Mutex,
+		(CPU_CHAR*) "RegisterOccupied_Mutex",
```

Comment:
Seeing as register isn't actually a great word to use in a starter project as it's confusing to people (again, my bad!) can you change the name of this mutex whenever it occurs to maybe something like Cashier_Mutex or something?